packages/google_sign_in/google_sign_in_web/lib/src/utils.dart

```diff
@@ -11,11 +11,15 @@ import 'generated/gapiauth2.dart' as auth2;
 
 /// Injects a bunch of libraries in the <head> and returns a
```

Comment:
Fix to reference `[target]`?

Reply:
Yep, great suggestion! I've improved the docs of this method a little bit.